server/controllers/index.js

```diff
@@ -0,0 +1,224 @@
+//module to contain the controller functions for postgres and cassandra
+
+//import the client and cassClient from server/index.js
+// const { pgClient, cassClient } = require('../index.js');
+
+
+//Require postgres
+const { Pool } = require('pg');
+
+//Create connection to postgres db
+const pgClient = new Pool({
+  user: 'jasonschreiber',
+  host: 'localhost',
+  database: 'titleservice',
+  password: 'password',
+  port: 5432
+});
+
+pgClient.connect()
+  .then(() => console.log('Postgres Database connected'))
+  .catch(() => console.log('Error connecting to db'));
+
+
+
+//Reauire cassandra
+var cassandra = require('cassandra-driver');
+
+//Create connection to cassandra db
+//Replace Username and Password with your cluster settings
+var authProvider = new cassandra.auth.PlainTextAuthProvider('cassandra', 'cassandra');
+//Replace PublicIP with the IP addresses of your clusters
+var contactPoints = ['127.0.0.1:9042'];
+//establish what localDataCenter to look for
+var localDataCenter = 'datacenter1';
+var cassClient = new cassandra.Client({contactPoints: contactPoints, localDataCenter: localDataCenter, authProvider: authProvider, keyspace:'sdc'});
+
+cassClient.connect(() => {
+  console.log('Cassandra Database connected');
+});
+
+
+module.exports = {
+  title: {
+    post: async function (req, res) {
+      //create variables for the id and title
+      const id = req.body.id;
+      const title = req.body.title;
+
+      //if the db is postgres
+      if (process.env.ENV_DB === 'pg') {
+        //take care of duplicates - if the id in the req exists already
+        try {
+          var query = await pgClient.query(`SELECT title FROM titles WHERE id = ${id}`);
```

Comment:
This looks like it'll introduce an SQL injection vulnerability. Be careful to avoid executing user input as if it were code. I'd say use a parameterized query instead as described here: https://node-postgres.com/features/queries#parameterized-query